content/en/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -45,19 +45,19 @@ kubectl get pods --output=yaml
 
 The output shows that the Pod owner is a ReplicaSet named `my-repset`:
 
-```shell
+```yaml
 apiVersion: v1
 kind: Pod
 metadata:
-  ...
+  …
```

Comment:
hello @sftim. I am not sure if the change on line 52 is needed. Have a look at the preview.

Reply:
I think I could leave that change out; for me, it's an improvement to the page.

Reply:
I agree with Karen that the long form is actually easier to read and would prefer reverting this particular change. At least, it's easier to read for my old eyes! :)

Reply:
Tweaked